src/components/form/Form.styeld.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const InputContainer = styled.div`
+	background-color: #f0f0f0;
+	width: 300px;
+	height: 620px;
+	display:flex: center;
+	padding-left: 1.8rem;
+	padding-top:20px;
```

Comment:
I'd decide on one type of unit, preferably relative ones like `rem`.

Reply:
is changed